optaplanner-docs/src/modules/ROOT/pages/quickstart/quarkus/quarkus-quickstart.adoc

```diff
@@ -86,16 +86,40 @@ $ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
 $ cd optaplanner-quickstart
 ----
 
-In Maven, your `pom.xml` file contains these dependencies:
+If you choose Maven, your `pom.xml` file has the following content:
 
 [source,xml,subs=attributes+]
 ----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>optaplanner-quarkus-school-timetabling-quickstart</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <maven.compiler.release>11</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <version.io.quarkus>{quarkus-version}</version.io.quarkus>
+    <version.org.optaplanner>{project-version}</version.org.optaplanner>
```

Comment:
`project-version` very much reminds `project.version`, which in this case would be `1.0-SNAPSHOT`. Could we call it `optaplanner-version` ?

Reply:
Agreed, but that needs to happen across all docs, so let's do that in a separate PR. I created a jira to track:
https://issues.redhat.com/browse/PLANNER-2570

Reply:
Note that {project-version} is nothing new. It's not introduced in this PR.